docs/agent-quickstart.md

```diff
@@ -68,22 +68,26 @@ To test your agent, you have two options:
 1. Run it on your local machine using the `fixie agent serve` command
 1. Deploy it to the Fixie platform using the `fixie agent deploy` command.
 
+Note that with either option you still need to use the Fixie Platform (e.g. https://app.fixie.ai/agents) to access your agent, but with `serve` you can
```

Comment:
This can be confusing. You can use any access method to the agent - REST API, GraphQL API, app.fixie.ai web app.

Reply:
Thanks for clarifying. Updated, PTAL.